compair/static/modules/assignment/assignment-form-partial.html

```diff
@@ -334,6 +330,81 @@ <h1 ng-if="method == 'copy'">Duplicate: {{originalAssignment.name}}</h1>
                 for this assignment.
     </fieldset>
 
+    <fieldset>
+        <legend>Self-Evaluation Settings</legend>
+        <div class="instructional-text">
+            <p>If enabled, students will also be required to self-evaluate by providing feedback on their own answer (after answering and comparing).</p>
```

Comment:
`(after answering and comparing)` isn't necessarily true (when date set). not sure how to reword better though

Reply:
I'll just remove "after comparing"